pkgs/applications/networking/instant-messengers/pantalaimon/default.nix

```diff
@@ -63,6 +63,12 @@ buildPythonApplication rec {
     pytest
   '';
 
+  postInstall = ''
+    install -Dm 0644 docs/man/pantalaimon.5 $out/share/man/man5/pantalaimon.5
+    install -Dm 0644 docs/man/pantalaimon.8 $out/share/man/man8/pantalaimon.8
+    install -Dm 0644 docs/man/panctl.1 $out/share/man/man1/panctl.1
```

Comment:
Please use installManPage.

Reply:
Didn't know that one yet - using it now.